Tasks/SonarQubePostTest/task.json

```diff
@@ -12,7 +12,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 42
+    "Patch": 43
```

Comment:
Since this is a completed feature, I am upping the version numbers so that the new tasks will be used